### PR TITLE
[ST] - IPv6 support for Kind on TF

### PR DESCRIPTION
**Pull request by see-quick**
### Type of change

- Enhancement / new feature
- Refactoring

### Description

This PR adds a support for running Kind (Kubernetes in docker) clusters using IPv6.
### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/main/java/io/strimzi/systemtest/Environment.java [hunk 1]
```diff
@@ -301,6 +302,10 @@ public static boolean isIpv4Family() {
         return IP_FAMILY.contains(IP_FAMILY_DEFAULT);
     }
 
+    public static boolean isIpv6Family() {
+        return IP_FAMILY.contains(IP_FAMILY_DEFAULT);
```
Comment (see-quick) [inline systemtest/src/main/java/io/strimzi/systemtest/Environment.java:306]: Here should be `IP_FAMILY_VERSION_6` :) copy-paste (I will update it when the acceptance testing farm will be done)

**Review by im-konge — APPROVED** [review]

File: systemtest/src/test/java/io/strimzi/systemtest/security/PodSecurityProfilesST.java [hunk 2]
```diff
@@ -92,6 +93,7 @@ public class PodSecurityProfilesST extends AbstractST {
      *  - security-profiles
      */
     @Tag(ACCEPTANCE)
+    @KindIPv6NotSupported("Kafka Connect Build using the Kaniko builder is not available under the restricted security profile")
```
Comment (see-quick) [inline systemtest/src/test/java/io/strimzi/systemtest/security/PodSecurityProfilesST.java:96]: This should be fixed in another PR :)
